cookbooks/aws-parallelcluster-slurm/recipes/update_shared_storages.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: update_shared_storages
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+load_shared_storages_mapping
+
+ruby_block "get volumes to mount and unmount" do
```

Comment:
The block title refers to volumes but the logic is not limited to EBS volumes, as it also covers EFS and FSx.
Let's use a more general title like "get storage to mount and unmount"